libs/core/src/lib/datetime/fd-date.ts

```diff
@@ -31,6 +32,11 @@ export class FdDate {
      */
     second: number;
 
+    /**
+     * Validation state
+     */
+    private _isValid: boolean;
```

Comment:
@hidden?

Reply:
@dimamarksman @InnaAtanasova Should we add "hidden" to private properties? Some time ago I had a talk with @Betrozov and he told me the we add hidden for public properties which we want to mark that it uses only inside controller and template.  Perhaps I misunderstood it. Could you please describe cases when we should use @hidden?

Reply:
We use @hidden annotation if we want to avoid its appearing in the public documentation.
properties that start with `_` are considered as private but in order to make it accessible from a template we are forced to keep it public